judgments/views/detail.py

```diff
@@ -78,10 +78,18 @@ def detail(request, judgment_uri):
 
     context = {}
 
-    if "press-summary" in judgment_uri:
+    press_summary_suffix = "/press-summary/1"
+    if judgment_uri.endswith(press_summary_suffix):
```

Comment:
I think this logic should probably be on the Judgment class.

Reply:
yep, 100% agree. Was planning on making a card to expose a relationship between the 2 in the api client. Needs a bit more thinking on the best way